docs/compute-element/htcondor-ce-overview.md

```diff
@@ -47,25 +47,25 @@ Benefits of running the HTCondor-CE:
     variables, which means that customizations will persist across upgrades and will not involve modification of
     software internals to route jobs
 
-How Jobs Run
-------------
+How CE Jobs Run
+---------------
 
-Once an incoming grid job is authorized, it is placed into HTCondor-CE’s scheduler where the JobRouter creates a
+Once an incoming CE job is authorized, it is placed into HTCondor-CE’s scheduler where the JobRouter creates a
 transformed copy (called the *routed job*) and submits the copy to the batch system (called the *batch system job*).
-After submission, HTCondor-CE monitors the batch system job and communicates its status to the original grid job, which
+After submission, HTCondor-CE monitors the batch system job and communicates its status to the original CE job, which
 in turn notifies the original submitter (e.g., job factory) of any updates.
 When the job completes, files are transferred along the same chain: from the batch system to the CE, then from the CE to
 the original submitter.
 
 ### Hosted CE over SSH
 
-The Hosted CE is intended for small sites or as an introduction to the OSG.
-The OSG configures and maintains an HTCondor-CE on behalf of the site.
+The Hosted CE is intended for small sites or as an introduction to providing capacity to collaborations.
+OSG staff configure and maintain an HTCondor-CE on behalf of the site.
 The Hosted CE is a special configuration of HTCondor-CE that can submit jobs to a remote cluster over SSH.
-It provides a simple starting point for opportunistic resource owners that want to start contributing to the OSG with
-minimal effort: an organization will be able to accept OSG jobs by allowing SSH access to a login node in their cluster.
+It provides a simple starting point for opportunistic resource owners that want to start contributing to collaborations with
```

Comment:
Why not just the approved phrase of “… contributing capacity …”?

Reply:
Fixed